Cubical/Codata/M/AsLimit/stream.agda

```diff
@@ -1,11 +1,16 @@
-{-# OPTIONS --cubical --guardedness --safe #-}
+{-# OPTIONS --cubical --guardedness #-}
```

Comment:
Why no safe?

Reply:
Oh, I should be safe, it is just easier to work with, without that flag.